dashboard/test/factories/factories.rb

```diff
@@ -797,6 +797,7 @@
   end
 
   factory :resource do
+    association :course_version, factory: :course_version
```

Comment:
nit: since the factory name matches, can this just be `association :course_version`?